server/halomod_app/__init__.py

```diff
@@ -0,0 +1,335 @@
+from flask import Flask, url_for, redirect, jsonify, request, session, abort
+from . import utils
+import base64
+from halomod import TracerHaloModel
+import json
+import dill as pickle
+import codecs
+import hmf
+from flask_cors import CORS
+import jsonpickle
+import time
+import redis
+from flask_session import Session
+from werkzeug.exceptions import InternalServerError, HTTPException
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+
+sess = Session()
+
+
+def create_app(test_config=None):
```

Comment:
is there a good reason to have all of this in a function? I haven't written flask stuff before, but that seems like a weird design

Reply:
Haha, this refactor has been on the to-do list for a while. We will eventually have related functionality grouped in separate files and leave endpoint stuff out of __init__ so its properly just a startup script.

Reply:
OK cool.